core/auth/src/main/java/software/amazon/awssdk/auth/token/credentials/TokenUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.auth.token.credentials;
+
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.identity.spi.TokenIdentity;
+
+@SdkInternalApi
```

Comment:
CredentialsUtils is `@SdkProtectedApi` since it is used in other modules, but this class isn't, so keeping this `@SdkInternalApi`. Once more SRA work is done, these conversion may become unnecessary and I'd like to delete this then.

Reply:
Can we have a TODO on the class to not forget the intent?
Since the class is internal can you move it to the `internal` package?